### PR TITLE
feat(bigquery): Add call to action to type change error message

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Same as https://github.com/cloudquery/cloudquery/pull/6643 not from a fork.

Similar to https://github.com/cloudquery/cloudquery/pull/6612, this PR adds a call to action when we change field types and makes the error message more consistent with other destinations

### CODE REVIEW

File: plugins/destination/bigquery/client/migrate.go [hunk 1]
```diff
@@ -178,10 +178,10 @@ func mergeSchemas(haveSchema, wantSchema bigquery.Schema) (bigquery.Schema, erro
 	for _, f := range haveSchema {
 		if want, ok := wantMap[f.Name]; ok {
 			if want.Type != f.Type {
-				return nil, fmt.Errorf("field %v changed type from %v to %v", f.Name, f.Type, want.Type)
+				return nil, fmt.Errorf("column %v changed type from %v to %v. Try dropping the column and re-running", f.Name, f.Type, want.Type)
 			}
 		} else if f.Required {
```
Comment (erezrokah) [inline plugins/destination/bigquery/client/migrate.go:183]: Do we need this `required` check? cc @hermanschaaf

Comment (hermanschaaf) [reply plugins/destination/bigquery/client/migrate.go:183]: Yes, if a required column is not in our schema, all inserts will fail

Comment (erezrokah) [reply plugins/destination/bigquery/client/migrate.go:183]: Are we setting it or is this something the user does?

Comment (hermanschaaf) [reply plugins/destination/bigquery/client/migrate.go:183]: It's something a user might do, so we're basically checking that they didn't do that.

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**